src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -105,7 +105,7 @@ case class CsvRelation protected[spark] (
     tokenRdd(schemaFields.map(_.name)).flatMap { tokens =>
 
       if (dropMalformed && schemaFields.length != tokens.size) {
-        logger.warn(s"Dropping malformed line: ${tokens.mkString(",")}")
+        logger.warn(s"Dropping malformed line (wrong length): ${tokens.mkString(",")}")
```

Comment:
Sorry for the delay - I've fixed those other messages, but here is it not useful to know which line has caused the error.  Compared to the errors below it's probably easier to find this in the original files, however it's still even easier if we tell the user about the problems lines :-)